trace/opencensus/trace/exporters/transports/background_thread.py

```diff
@@ -158,11 +155,14 @@ def stop(self):
             return True
 
         with self._lock:
+            print('Number of spans pending sent: {}'.format(
```

Comment:
Is the printing meant to stay or is it just for debugging?

If intentional, should it be some kind of optional, configurable logger instead?

Reply:
Removed.